### PR TITLE
add 'copy' function to filetools for easy copying of lists of files/directories

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/tools/filetools.py [hunk 1]
```diff
@@ -1455,3 +1457,29 @@ def copy_dir(path, target_path, force_in_dry_run=False):
...
+    :param force_in_dry_run: force running the command during dry run
+    """
+    if isinstance(paths, basestring):
+        paths = [paths]
+
+    _log.info("Copying %d files & directories to %s", len(paths), target_path)
+
+    for path in paths:
+
+        full_target_path = os.path.join(target_path, os.path.basename(path))
```
Comment (wpoely86) [inline easybuild/tools/filetools.py]: `target_path` (in the last part)?

File: easybuild/tools/filetools.py [hunk 2]
```diff
@@ -1455,3 +1457,29 @@ def copy_dir(path, target_path, force_in_dry_run=False):
...
+    :param target_path: target location
+    :param force_in_dry_run: force running the command during dry run
+    """
+    if isinstance(paths, basestring):
+        paths = [paths]
+
+    _log.info("Copying %d files & directories to %s", len(paths), target_path)
+
+    for path in paths:
+
```
Comment (wpoely86) [inline easybuild/tools/filetools.py]: less whitespace?

File: easybuild/tools/filetools.py (hunk 1 above)
Comment (boegel) [inline easybuild/tools/filetools.py]: No, `target_path` is the directory we're copying to, `full_target_path` is the path that also includes the name of the file/directory we're copying to...

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**